entrypoint.d/toggle-profiler-on-startup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+if [ "${PHP_DEBUGGER}" = 'xdebug' ]; then
```

Comment:
```suggestion
if [[ ! -z "${PHP_DEBUGGER}" ]] && [[ "$PHP_DEBUGGER" != 'xdebug' ]]; then
```

Reply:
https://explainshell.com/explain?cmd=if+%5B%5B+%21+-z+%22%24%7BPHP_DEBUGGER%7D%22+%5D%5D+%26%26+%5B%5B+%22%24PHP_DEBUGGER%22+%21%3D+%27xdebug%27+%5D%5D%3B+then

Reply:
how about now?

Reply:
now it should work ;)